data/arabic/live/cvp5r6m6mgpt.json

```diff
@@ -0,0 +1,324 @@
+{
+  "data": {
+    "title": "نتائج الانتخابات الأمريكية: احصل على آخر التحديثات",
+    "description": "البث المباشر: حفل تنصيب الرئيس الخامس والأربعين للولايات المتحدة",
+    "language": "ar",
+    "headerImage": null,
+    "promoImage": null,
+    "home": "arabic",
+    "section": null,
+    "commercialInfo": {
+      "adCategory": null,
+      "adSubCategory": null,
+      "hasAdvertising": false
+    },
+    "sportDataEvent": {
+      "id": null
+    },
+    "eavisEvent": {
+      "id": null
+    },
+    "article": {
+      "id": null
+    },
+    "mediaCollections": [
+      {
+        "type": "liveMedia",
+        "model": {
+          "urn": "urn:bbc:pips:sid:bbc_arabic_tv",
+          "title": "BBC Arabic TV",
+          "type": "episode",
+          "synopses": {
+            "short": "جولة إخبارية يومية تتناول أهم الأحداث العربية والعالمية في تقارير ولقاءات وتحليلات ",
+            "medium": "جولة إخبارية يومية تتناول أهم الأحداث العربية والعالمية في تقارير ولقاءات وتحليلات ",
+            "long": "جولة إخبارية يومية تتناول أهم الأحداث العربية والعالمية في تقارير ولقاءات وتحليلات "
+          },
+          "mediaType": "audio_video",
+          "imageUrlTemplate": "https://ichef.bbci.co.uk/images/ic/$recipe/p08b23t4.png",
+          "masterbrand": {
+            "id": "bbc_arabic_tv",
+            "name": "تلفزيون بي بي سي عربي",
+            "networkName": "تلفزيون بي بي سي عربي",
+            "type": "tv",
+            "imageUrlTemplate": "ichef.bbci.co.uk/images/ic/$recipe/p08b23t4.png"
+          },
+          "version": {
+            "vpid": "n4pdm3cdh4",
+            "duration": "PT1H",
```

Comment:
This is what was returned from the live-media-collection API - it appears to be the last broadcast programme on the channel...